app/views/users/show.html.erb

```diff
@@ -1 +1,35 @@
-<h1>This page shows single user by specific id</h1>
\ No newline at end of file
+<main>
+    <div class="single-user-container">
+        <div class="single-user">
+            <div class="user-image">
+                <img src="<%= @user.photo%>" alt="profile" class="user-profile">
+            </div>
+            <div class="user-information">
+                <h2 class="user-name"><%= @user.name%></h2>
+                <span class="post-counter"><p class="count">Number of posts: <%= @user.posts.count %></p></span>
+            </div>
+        </div>
+        <div class="user-bio">
+            <h4 class="bio">Bio</h4>
+            <p class="bio-paragraph">
+                <%= @user.bio %>
+            </p>
+        </div>
+        <div class="user-all-posts">
+            <% @user.posts.each_with_index do |post,index|%>
+            <div class="user-post">
+                <h4 class="post">Post #<%= index+1%></h4>
+                <div class="post-content">
+                    <p class="post-paragraph">
+                        <%= post.text %>
+                    </p>
+                    <span class="posts-count"><p class="count">Comments: <%=post.comments_counter%>, Likes: <%=post.likes_counter%></p></span>
+                </div>
+            </div>
+            <%end%>
+        </div>
+        <div class="see-all">
+            <button class="all">See all posts</button>
```

Comment:
- [ ] As per the requirement, this button must redirect to the user post index page. Kindly add this.